redirects-list.json

```diff
@@ -38,19 +38,19 @@
   "^/doc/install/plugins(/.*)?$                                                           /doc/install/ide-plugins$1",
 
   "^/doc/get-started(/.*)?$                                                               /doc/start",
-  "^/doc/start/data-versioning$                                                           /doc/start/data-and-model-versioning",
-  "^/doc/start/data-access$                                                               /doc/start/data-and-model-access",
-  "^/doc/start/data-and-model-versioning(/.*)?$                                           /doc/start/data-management$1 302",
-  "^/doc/start/data-and-model-access(/.*)?$                                               /doc/start/data-management/access$1 302",
-  "^/doc/start/data-pipelines(/.*)?$                                                      /doc/start/data-management/pipelines$1 302",
-  "^/doc/start/metrics-parameters-plots(/.*)?$                                            /doc/start/data-management/metrics-parameters-plots$1 302",
-  "^/doc/start/sharing-data-and-model-files$                                              /doc/start/data-and-model-versioning#storing-and-sharing 302",
+  "^/doc/start/data-versioning$                                                           /doc/start/data-management",
+  "^/doc/start/data-and-model-versioning(/.*)?$                                           /doc/start/data-management 302",
+  "^/doc/start/sharing-data-and-model-files$                                              /doc/start/data-management#storing-and-sharing 302",
+  "^/doc/start/data-access$                                                               /doc/start/data-management/data-and-model-access",
+  "^/doc/start/data-and-model-access(/.*)?$                                               /doc/start/data-management/data-and-model-access 302",
+  "^/doc/start/data-pipelines(/.*)?$                                                      /doc/start/data-management/data-pipelines 302",
+  "^/doc/start/metrics-parameters-plots(/.*)?$                                            /doc/start/data-management/metrics-parameters-plots 302",
```

Comment:
Updated and simplified redirects as well (using final locations for previous 301 ones, plus fortunately many were 302s so we can change those any time).